### DISCUSSION TITLE
Incubator.Toast - render null if no icon prop provided

### DISCUSSION THREAD

**Discussion by Waryen** [wix/react-native-ui-lib#2189] [Ideas]
Hello!

I have a suggestion to modify the toast component, I want to add the possbility to only render the message without having a blank space on the left because there is no icon provided.

I have forked the project and modified the toast component to not return the icon if there are no icon prop, [check it out here](https://github.com/Waryen/react-native-ui-lib/commit/4ed52ccf1ea18b124ec479c7ac95a03e5abede76)

What do you think about it?

**Comment by ethanshar** [comment]
Hi @Waryen 
It looks good. 
Can you submit a PR so we can review and merge!

**Reply by Waryen** [reply]
Hi, thanks for your reply, I've [opened a PR](https://github.com/wix/react-native-ui-lib/pull/2263)
